src/backend/dialog/dialog.ts

```diff
@@ -37,12 +33,18 @@ function showDialogBoxModalAuto(props: {
       logWarning(['showDialogBoxModalAuto:', error], {
         prefix: LogPrefix.Backend
       })
+
+      const window = getMainWindow()
+
       switch (props.type) {
         case 'ERROR':
           showErrorBox(props.title, props.message)
           break
         default:
-          showMessageBox(BrowserWindow.getAllWindows()[0], {
+          if (!window) {
+            break
+          }
```

Comment:
we were not checking this before and it could fail if for some reason we call this without a window

Reply:
I thought all electron dialogs were replaced with custom once?

Reply:
We forgot a few, at least two of them are still using the old one.

To be fair, I think because of things like the Steam deck, we should keep at least the one related to the game not found as native, because otherwise, Heroic fails. For now it simply returns to steam, but the Game needs to update one (that is native now), was making Steam completely freeze since it was waiting for the user input.

another idea is to extend these methods and make them accept a parameter like: `nativeDialog: boolean`, so when using NO-GUI we can specify which ones are native or not.